### ISSUE TITLE
PeoplePicker - NormalPeoplePicker - Android talkback incorrect focus order when trying to select a suggestion

### ISSUE THREAD

**Issue by kestill** [microsoft/fluentui#17038]
### Environment Information

- **Package version(s)**: 7.158.3
- **Browser and OS versions**: Android OS 10, Pixel 3a/Chrome Browser 86.0.4240.198

### Please provide a reproduction of the issue in a codepen:

1. https://codepen.io/kestill/pen/dyOWEKg
2. Press textbox to open suggestions menu
3. Swipe right and observe talkback focus

#### Actual behavior:

After suggestions are shown, swipe right then talkback focus will land on first element of Chrome web page.

#### Expected behavior:

After suggestions available when user swipe right focus should be on 'first suggestion' and Talkback should announce 'first suggestion'.

### Documentation describing expected behavior

MAS Reference: [MAS 2.4.3 - Focus Order](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7b0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7d)

**Comment by gouttierre** [comment]
@kestill - thanks for the feedback and codepen share.

@khmakoto - Would you be able to confirm if this is a regression, or if this behavior is an issue with Talkback on Android OS?
